aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -177,7 +177,11 @@ Tensor& set_storage_quantized_(
   auto* self_ = self.unsafeGetTensorImpl();
   self_->set_storage_keep_dtype(std::move(storage));
   self_->set_storage_offset(storage_offset);
-  self_->set_sizes_and_strides(sizes, strides);
+  if(strides.data() == nullptr){
+    self_->set_sizes_contiguous(sizes);
+  }else{
+    self_->set_sizes_and_strides(sizes, strides);
```

Comment:
why is this change needed?

Reply:
hi @kimishpatel ,This modification is for the functionality of pickling non-cpu QTensor. 
  
Refer to https://github.com/pytorch/pytorch/pull/108400/files#diff-01ceffff32cac1db5af88e894b595f6e969d2db30b62f4e10842f336a7ca01e3L729, non-cpu Tensor is saved through putting it on host firstly. (`at::empty({0}, tensor.options()).set_(tensor.storage(), 0, size, {1}).cpu()`). 

 We add `at::empty_quantized({0}, tensor.options()).set_(tensor.storage(), 0, size, {1}).cpu()` for QTensor. 

`set_` would pass an empty stride for `set_storage_quantized`(https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/TensorShape.cpp#L373). But, the `set_storage_quantized` cannot handle an empty `strides`. This line modification is for QTensor to handle the case with empty strides (by treat it as an contiguous tensor).